### PR TITLE
Only validate CSV headers when CSV has headers

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
### Checklist

- [ ] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [ ] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [ ] The description lists all applicable issues this PR seeks to resolve
- [ ] The description lists any configuration setting(s) that differ from the default settings
- [ ] All tests and CI builds passing
- [ ] The description lists all relevant PRs included in this release _(remove this if not merging to master)_
- [ ] e2e tests are all passing _(remove this if not merging to master)_

### Description

CSV Headers are not present when appending. We should not validate them as such.

### CODE REVIEW

**Review by amy-corson-ibigroup — APPROVED** [review]
Seems like a good rule of thumb when validating CSV headers

File: lib/manager/components/transform/CustomCSVForm.js [hunk 1]
```diff
@@ -34,9 +35,11 @@ const CustomCSVForm = (props: Props) => {
   } = props
 
   useEffect(() => {
+  // Default to true
+    const validateHeaders = props.validateHeaders !== undefined ? props.validateHeaders : true
```
Comment (josh-willis-arcadis) [inline lib/manager/components/transform/CustomCSVForm.js:39]: Do ES Lint rules apply to this repo? I know there are rules for a multiline ternary.

**Review by josh-willis-arcadis — COMMENTED** [review]
If ES Lint rules do not apply then all looks good to me.

File: lib/manager/components/transform/CustomCSVForm.js (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/manager/components/transform/CustomCSVForm.js:39]: > Do ES Lint rules apply to this repo? I know there are rules for a multiline ternary.

Datatools has its own eslint config, supplied by [mastarm](https://github.com/conveyal/mastarm)

**Review by josh-willis-arcadis — APPROVED** [review]

**Pull request merged**